.browserslistrc

```diff
@@ -10,9 +10,11 @@
 
 last 1 Chrome version
 last 1 Firefox version
-last 2 Edge major versions
-last 2 Safari major versions
+last 1 Edge major versions
+last 1 Safari major versions
 last 2 iOS major versions
```

Comment:
Doesn't apple release a new version of Safari and iOS every year?
Why do we support iOS longer than Safari?

Reply:
I went by the general rule “desktop 1 last version, mobile 2 last versions”.